### PR TITLE
India - LGD - States Data

### PR DESCRIPTION
**Pull request by thejeshgn**
@pradh 

- Added states data.
- [CSV and TMCF Browser](https://datacommonsorg.github.io/tools/mcf-browser/#&file=https://raw.githubusercontent.com/datacommonsorg/data/2a6de8fcfd1584c672402229698d93d03e67ef45/scripts/india_lgd/local_government_directory_states/LocalGovernmentDirectory_States.tmcf&file=https://raw.githubusercontent.com/datacommonsorg/data/2a6de8fcfd1584c672402229698d93d03e67ef45/scripts/india_lgd/local_government_directory_states/LocalGovernmentDirectory_States.csv)

### CODE REVIEW

File: scripts/india_lgd/local_government_directory_states/preprocess.py [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+
+    def process(self):
+        # Load the lgd states data and set the type of columns to str
+        # if there are NA values then replace it with '' character
+        self.lgd_df = pd.read_csv(self.lgd_csv, dtype=str)
+        self.lgd_df.fillna('', inplace=True)
+        # Drop title rows in the top and empty rows after 39.
+        # The actual data is between 2nd and 40th row. So keep only them.
+        self.lgd_df = self.lgd_df.iloc[1:38]
+        # Take the the header row and set it has column header
```
Comment (pradh) [inline scripts/india_lgd/local_government_directory_states/preprocess.py:70]: nit: as

File: scripts/india_lgd/local_government_directory_states/preprocess.py [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+                              ["of", "and"] else name.capitalize())
+        return " ".join(first_list)
+
+    @staticmethod
+    def format_code(s):
+        # Converts into two character length code
+        # If the value is `0` then it makes it empty
+        # If the length is single character then it prepends it
+        # with `0` to make it two character length
+        if len(s) == 1:
```
Comment (pradh) [inline scripts/india_lgd/local_government_directory_states/preprocess.py:50]: would zfill work?

```
  s = s.zfill(2)
  return "" if s == "00" else s
```

**Review by pradh — APPROVED** [review]

File: scripts/india_lgd/local_government_directory_states/preprocess.py (hunk 1 above)
Comment (thejeshgn) [reply scripts/india_lgd/local_government_directory_states/preprocess.py]: 👍🏼

File: scripts/india_lgd/local_government_directory_states/preprocess.py (hunk 2 above)
Comment (thejeshgn) [reply scripts/india_lgd/local_government_directory_states/preprocess.py]: 👍🏼

**Review by pradh — APPROVED** [review]
Thanks for the update.

**Pull request merged**